dockers/docker-platform-monitor/docker-pmon.supervisord.conf.j2

```diff
@@ -183,3 +183,16 @@ startsecs=10
 dependent_startup=true
 dependent_startup_wait_for=rsyslogd:running
 {% endif %}
+
+{% if not skip_sensormond %}
```

Comment:
what will be behaviour on platfrom where `skip_sensormond` is not define ? Will it cause any issue ?

@bmridul

Reply:
If skip_sensormond is not defined, sensormond will run inside PMON docker. 

If the platform has not implemented the APIs for providing voltage and current sensors, sensormond will not do anything as it will retrieve an empty list for the voltage and current sensors. The show commands will display that no sensors are detected. This is what I got on Cisco platform when I removed the platform support for sensormond.

cisco@sonic:~$ docker exec pmon ps aux | grep sensormond
root          36  0.7  0.1  49872 36996 pts/0    S    14:08   0:03 /usr/bin/python3 /usr/local/bin/sensormond
cisco@sonic:~$ show platform voltage
Sensor not detected
cisco@sonic:~$ show platform current
Sensor not detected

I did not observe any issues.

Reply:
@bmridul can we please keep the default to NOT run sensormond. All platforms except Cisco will be running sensordmond and in future we can revisit if needed.

Reply:
Fixed.